modules/core/04-channel/types/packet.go

```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/cosmos/ibc-go/modules/core/exported"
 )
 
+// NewPacketId returns a new instance of PacketId
```

Comment:
I think @colin-axner will likely flag this as needing to be a separate PR into main with a changelog. We should probs add the validation fn here as well.

Reply:
I'll move it over in my validation PR for now as well

Reply:
Actually, disregard that entirely. For some reason, I thought we made this change on main and not in the fee middleware branch -> https://github.com/cosmos/ibc-go/pull/272/files#diff-fc6caa5ba0a543336f3c02d081f09660a36e55412cca742e619e94ba959bf4c5R135

I must have been mixing this up with another change